docs/CONTRIBUTING.md

```diff
@@ -54,6 +54,10 @@ submitting a PR:
 - PR titles indirectly become part of the CHANGELOG so it's crucial to provide a good record of **what** change is being
   made in the title; **why** it was made will go in the PR description, along with a link to a GitHub issue if it
   exists.
+- <a id="breaking"></a>A breaking change is a change that forces users to change their code, command-line invocations,
+  build scripts or data files when upgrading from an older version of arduino-cli. If the PR contains a breaking change,
+  please start the commit message and PR title with the string **[breaking]**. Don't forget to describe what changes
+  might users need to make in their workflow or application due to this PR
```

Comment:
```suggestion
- <a id="breaking"></a> If the PR contains a breaking change, please start the commit message and PR title with the
  string **[breaking]**. Don't forget to describe what changes users might need to make in their workflow or application
  due to this PR. A breaking change is a change that forces users to change their code, command-line invocations, build
  scripts or data files when upgrading from an older version of Arduino CLI.
```
I think this wording improves readability because it provides the condition up front, so it's easier to understand the purpose of the bullet point when skimming the document. That makes it easier for those who need it to find, and faster for those who don't need it to skip over.

---
I also made the change from `arduino-cli` -> "Arduino CLI" as in my suggestion above. so if I'm wrong about the semantics, that part of the change must be reverted.